pkg/kubelet/secret/secret_manager.go

```diff
@@ -67,6 +69,8 @@ func (s *simpleSecretManager) RegisterPod(pod *v1.Pod) {
 func (s *simpleSecretManager) UnregisterPod(pod *v1.Pod) {
 }
 
+type GetObjectTTLFunc func() (time.Duration, bool)
```

Comment:
@yujuhong - do you want to review the whole PR, or just the kubelet part and I should find someone else for the controller part?